mpsl/CHANGELOG.rst

```diff
@@ -49,9 +50,9 @@ Changes
 Bugfixes
 ========
 
-* Fixed an issue where the high frequency clock and ``TIMER0`` were not turned off during idle periods shorter than 9 ms (DRGN-14152).
-  This increased the average power consumption.
-  Such a case could occur when running a |BLE| connection with a connection interval of 7.5 ms.
+Fixed an issue where the high frequency clock and ``TIMER0`` were not turned off during idle periods shorter than 9 ms (DRGN-14152).
+This increased the average power consumption.
+Such a case could occur when running a |BLE| connection with a connection interval of 7.5 ms.
```

Comment:
We usually keep bullet point even just for one issue, to avoid formatting mismatch from section to section. 
See for example the built page: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/mpsl/CHANGELOG.html

This is a global change for this entire PR.

Reply:
I agree and would keep the bullet points as well, but it seemed there was a consensus to remove them in the sync meeting.

Reply:
I understood the consensus was about the single limitations entry. I don't agree removing bullet points for other cases is a good idea.